pkg/cmd/release/create/http.go

```diff
@@ -254,3 +272,20 @@ func deleteRelease(httpClient *http.Client, release *shared.Release) error {
 	}
 	return nil
 }
+
+func isOAuthToken(resp *http.Response) bool {
+	scopes := resp.Header.Get("X-Oauth-Scopes")
+	return scopes != ""
+}
+
+func oAuthTokenHasWorkflowScope(resp *http.Response) bool {
+	scopes := resp.Header.Get("X-Oauth-Scopes")
+
+	for _, s := range strings.Split(scopes, ",") {
+		if s == "workflow" {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
I would have loved for these functions to live somewhere else instead of locally in this command. I'm also not a fan of having two functions here. Open to feedback on ways to improve this situation.